src/java.base/share/classes/java/lang/System.java

```diff
@@ -471,6 +471,21 @@ public static native void arraycopy(Object src,  int  srcPos,
      * hashCode().
      * The hash code for the null reference is zero.
      *
+     * <div class="preview-block">
+     *      <div class="preview-comment">
+     *          The "identity hash code" of a {@linkplain Class#isValue() value object}
+     *          is computed by combining the hash codes of the value object's fields recursively.
```

Comment:
Doesn’t this work by recursively calling the `identityHashCode` algorithm?
```suggestion
     *          The "identity hash code" of a {@linkplain Class#isValue() value object}
     *          is computed by combining the identity hash codes of the value object's
     *          fields recursively.
```